Dockerfile

```diff
@@ -8,8 +8,8 @@ ADD https://beta.quicklisp.org/quicklisp.lisp .
 # Install Quicklisp and load FiveAM
 RUN sbcl --load quicklisp.lisp \
          --eval '(quicklisp-quickstart:install)' \
-         --eval "(ql:quickload '(:fiveam :dissect :st-json))" --quit
+         --eval "(ql:quickload '(:fiveam :st-json))" --quit
 # Copy over the test runner
-COPY bin/run.lisp /bin/run.sh bin/
+COPY bin/run.lisp bin/run.sh bin/
```

Comment:
Interesting that it worked with the leading slash.

Reply:
Yeah, it puzzled me too when I saw it... Fixed now though!